Rakefile

```diff
@@ -46,7 +46,7 @@ def normalize_bullets(markdown)
 end
 
 def linkify_prs(markdown)
-  markdown.gsub(%r!#(\d+)!) do |word|
+  markdown.gsub(%r!(?<=[^&])#(\d+)!) do |word|
```

Comment:
This should really be a *negative lookbehind*, but I couldn't get that to work. Instead, this is a *positive lookbehind* that asserts that the preceding character was anything other than a `&`.

If anybody with some RegExFoo can make this better, that would be lovely.